model/import/Parser/InvalidImportException.php

```diff
@@ -26,26 +26,33 @@
 
 class InvalidImportException extends Exception
 {
-    /** @var array */
-    private $errors;
-
-    /** @var int */
     private $totalErrors = 0;
+    private $messages = [];
 
-    public function addError(int $line, string $message): self
+    protected const LEVEL = 1;
```

Comment:
I think error level is more clear, LEVEL seems related to nested errors or its layer IMO.

```suggestion
    protected const ERROR_LEVEL = 1;
```